mkdocs.yml

```diff
@@ -0,0 +1,94 @@
+---
+site_name: Ansible Meetup Toolkit Documentation
+site_url: https://meetuptoolkit.readthedocs.io/
+repo_url: https://github.com/ansible/meetup
+edit_uri: blob/main/docs
+strict: true
+use_directory_urls: false
+
+theme:
+  name: "ansible"
+  features:
+    - content.code.copy
+    - content.action.edit
+    - navigation.expand
+    - navigation.sections
+    - navigation.instant
+    - navigation.indexes
+    - navigation.tracking
+    - toc.integrate
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      primary: teal
+      accent: blue
+      scheme: default
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      primary: teal
+      accent: blue
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+nav:
+  - Home: meetup_toolkit.md
```

Comment:
Can we change this to "index.md" or rearrange things so that the build produces an "index.html" file as the top-level page?

When I build this the root page is "meetup_toolkit.html" which isn't really the convention for most web servers AFAIK. I know we're using `use_directory_urls: false` to make it easy to navigate the site on local builds but I don't know if that makes any difference wherever this will be hosted (probably readthedocs).

Maybe it doesn't really matter but it feels weird to me. Also here is a brief article that explains "index.html" nicely: https://dpericich.medium.com/why-do-we-use-index-html-files-71cf1973e10

Reply:
yeah I debated that myself and went the lazy way. I'll rename the file or create some other index.md 

Thanks for taking a look!